prometheus-metrics/src/main/scala/org/http4s/metrics/prometheus/PrometheusExportService.scala

```diff
@@ -46,7 +46,7 @@ object PrometheusExportService {
 
   def service[F[_]: Sync](collectorRegistry: CollectorRegistry): HttpRoutes[F] =
     HttpRoutes.of[F] {
-      case req if req.method == Method.GET && req.pathInfo == "/metrics" =>
+      case req if req.method == Method.GET && req.pathInfo.toString == "/metrics" =>
```

Comment:
Alternatively, we could do
```Scala
req.pathInfo == path"/metrics"
```
which is something I've seen used in the codebase before.

Reply:
I like the `path` version, and we could store that to a constant in the service.

And if we used cats' `===`, the bug wouldn't have compiled.

Reply:
Done!